specification/applicationinsights/data-plane/Monitor.Exporters/preview/2020-09-15/swagger.json

```diff
@@ -0,0 +1,1044 @@
+{
+    "swagger": "2.0",
+    "info": {
+      "version": "2020-09-15_Preview",
+      "title": "ApplicationInsightsClient",
+      "description": "This document describes the protocol for client requests and responses to the data collection endpoint."
+    },
+   
+    "x-ms-parameterized-host": {
+      "hostTemplate": "{Host}/v2/",
+      "useSchemePrefix": false,
+      "parameters": [
+        {
+          "$ref": "#/parameters/Host"
+        }
+      ]
+    },
+    "paths": {
+      "/track": {
+        "post": {
+          "summary": "Detect anomalies for the entire series in batch.",
+          "description": "This operation generates a model using an entire series, each point is detected with the same model. With this method, points before and after a certain point are used to determine whether it is an anomaly. The entire detection can give user an overall status of the time series.",
+          "operationId": "track",
+          "parameters": [
+            {
+              "name": "body",
+              "in": "body",
+              "description": "Time series points and period if needed. Advanced model parameters can also be set in the request.",
+              "required": true,
+              "schema": {
+                  "type": "array",
+                  "items": {
+                    "$ref": "#/definitions/TelemetryEnvelope"
+                  }
+              }
+            }
+          ],
+          "consumes": [
+            "application/json",
+            "x-json-stream"
+          ],
+          "produces": [
+            "application/json"
+          ],
+          "responses": {
+            "200": {
+                "description": "All of the telemetry items were accepted and processed.",
+                "schema": {
+                    "$ref": "#/definitions/TrackResponse"
+                }
+            },
+            "206": {
+                "description": "Partial success. Some of the telemetry items were accepted and processed.",
+                "schema": {
+                    "$ref": "#/definitions/TrackResponse"
+                }
+            },
+            "400" : {
```

Comment:
One would think the non-success codes would return `ExceptionData` instead of `TrackResponse` -- I don't actually see where `ExceptionData` ever gets used?

Reply:
`ExceptionData` is one the telemetry event types - events to track exceptions. It's an input type.

`TrackResponse` is the output model that is returned for both success and error response as this is a batch call and partial success is represented by TrackResposne.